lib/perlfaq4.pod

```diff
@@ -926,7 +926,7 @@ implementing it yourself is highly recommended; you'll save yourself odd bugs
 popping up later by just using code which has already been tried and tested in
 production for years.
 
-=head2 How do I strip blank space from the beginning/end of a string?
+=head2 How do I trim a string to strip blank space from the beginning/end?
```

Comment:
If you are going to change the title, a better one would be "How do I trim a string". If you think they know what trim is, they don't need the rest of the words.

Reply:
But, also consider the people who haven't used other languages and haven't encountered trim.

Thinking about this, I think this change might wait for a `trim` to show up in Perl.

Reply:
I see this more as a combination of using the English verb "trim" with it coincidentally being the name of this common operation. Thus the additional explanation helps describe that it's referring to blank space specifically.

Reply:
I was shooting for keeping `perldoc -q` working, but to that end I would probably go with "remove space" as something people might search for.  But, overall, it seemed as though folks thought that "trim" was the most common spelling for doing this to a string.  I'm not much on SEO though, so whatever folks think is a good title.  I thought this might make perl 5.34 while I'm sure `trim` won't.

> How do I trim a string to remove whitespace from the beginning/end?